From d202b10ee8ca606682bd83b8206a9483537c0f45 Mon Sep 17 00:00:00 2001
From: Charles Short <charles.short@windriver.com>
Date: Sun, 30 Jan 2022 01:47:52 +0000
Subject: [PATCH] Remove time-daemon conflict

Remove the virtual dependency for linuxptp so it does not conflict
with ntp.

Signed-off-by: Charles Short <charles.short@windriver.com>
---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 0715b74..9ffbdb4 100644
--- a/debian/control
+++ b/debian/control
@@ -14,7 +14,6 @@ Vcs-Browser: https://salsa.debian.org/multimedia-team/linuxptp
 Package: linuxptp
 Architecture: linux-any
 Depends: ${shlibs:Depends}, ${misc:Depends}
-Provides: time-daemon
 Description: Precision Time Protocol (PTP, IEEE1588) implementation for Linux
  Linuxptp is an implementation of the Precision Time Protocol (PTP)
  according to IEEE standard 1588 for Linux. Features include:
-- 
2.32.0

